src/Adapters/ADT3DSceneAdapter.ts

```diff
@@ -42,7 +46,8 @@ export default class ADT3DSceneAdapter {
         uniqueObjectId?: string,
         adtProxyServerPath = '/proxy/adt',
         blobProxyServerPath = '/proxy/blob',
-        useProxy = true
+        useProxy = true,
```

Comment:
now, can we make it explicity useAdtProxy to make it more clear as we did for adtProxyServerPath and blobProxyServerPath?
```suggestion
        useAdtProxy = true,
```